src/test/java/learnositysdk/request/DataApiIT.java

```diff
@@ -196,9 +207,7 @@ public void assertConsistentResponseRemote(Remote responseRemote, JSONObject res
 	private void assertConsistentResponseCodeStatus(int statusCode, boolean status)
 		throws org.json.JSONException
 	{
-		assertTrue("Inconsistent HTTP status cand and meta status",
-				(statusCode == 200 && status)
-				|| (statusCode != 200 && !status));
+		assertTrue((statusCode == 200 && status) || (statusCode != 200 && !status), "Inconsistent HTTP status cand and meta status");
```

Comment:
It's probably good to keep this split across multiple lines to improve readability like:

```java
assertTrue(
    (statusCode == 200 && status) || (statusCode != 200 && !status),
    "Inconsistent HTTP status cand and meta status"
);
```